content/blog/ultimate-windows-development-environment-guide/index.md

```diff
@@ -404,6 +411,10 @@ If you prefer an alternative shell, such as ZSH or Fish, you can install those i
 
 To get the alternative shell running any time you call `bash`, you'll need to configure your `.bashrc` file. You're able to run `nano ~/.bashrc` to open the file. Once done, add `bash -c zsh` to the top of the file. After this, every time `bash` runs, it will open `zsh`.
 
+That said, there's something not quite right about using a program called `bash` to run a shell outside of `bash` itself. Additionally, you'll notice that if you type `exit` in `zsh`, it won't exit to the windows shell; instead, it'll bring you to `bash` and require one more `exit` before bringing you back to the shell from whence you came.
```

Comment:
```suggestion
That said, there's something a little awkward about using a program called `bash` to run a shell outside of `bash` itself. Additionally, you'll notice that if you type `exit` in `zsh`, it won't exit to the Windows shell. Instead, it will bring you to `bash` and require one more `exit` before you are back to the shell from whence you came.
```